services/QuillLMS/client/app/bundles/Teacher/components/premium/premium_features_table.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react'
+
+import { premiumFeatures, } from './premium_features_data'
+import InfoTooltip from './infoTooltip'
+
+const greenCheckSrc = `${process.env.CDN_URL}/images/icons/icons-check-green.svg`
+
+const greenCheckCell = (<span className="check-or-empty-wrapper">
+  <img alt="Check icon" src={greenCheckSrc} />
+</span>)
+
+const emptyCell = (<span className="check-or-empty-wrapper" />)
+
+const Row = ({ label, tooltipText, basic, teacher, school, }) => (
+  <div className="premium-features-table-row">
+    <span className="label-and-tooltip">
+      <p>{label}</p>
+      <InfoTooltip tooltipText={tooltipText} />
+    </span>
+    {basic ? greenCheckCell : emptyCell}
+    {teacher ? greenCheckCell : emptyCell}
+    {school ? greenCheckCell : emptyCell}
+  </div>
+)
+
+const premiumFeaturesComponents = ({ independentPracticeActivityCount, lessonsActivityCount, diagnosticActivityCount, }) => premiumFeatures({ independentPracticeActivityCount, lessonsActivityCount, diagnosticActivityCount, }).map(section => (
+  <React.Fragment key={section.header}>
+    <div className="header-row">
+      <h4>{section.header}</h4>
+    </div>
+    {section.features.map(feature => <Row key={feature.label} {...feature} />)}
+  </React.Fragment>
+))
```

Comment:
love the way you handled this! I would just maybe break the `premiumFeatures` return out onto the next line so it's a bit easier to read